component-interaction/src/app/age-user-pipe.ts

```diff
@@ -0,0 +1,9 @@
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({ name: "ageUsers" })
+export class AgeUserPipe implements PipeTransform {
+
+	public transform(date: any, ...args: any[]): number {
```

Comment:
```suggestion
	public transform(dateOfBirth: Date, ...args: any[]): number {
```
new Date(). and date. was confusing for me